syntax = "proto3";

package Entities.Protobuf.Proto;

enum Status {
    STATUS_NEW = 0;
    STATUS_TO_PROVIDER = 1;
    STATUS_FILLED = 2;
    STATUS_PARTIAL_FILLED = 3;
    STATUS_REJECTED = 4;
}